README.md

```diff
@@ -37,6 +55,55 @@ $userConsent->setUsage('E-mail marketing campaigns');
 $userConsent->setUsageFieldName('gdpr_consent_usage');
 
 $subscriber->subscribe('hello@super.test', ['Name' => 'John Doe'], [$userConsent]);
+
+/**
+ * Method for sending transactional emails (YMLP does not support transactional emails).
+ *
+ * @param array<\SubscribeMe\ValueObject\EmailAddress> $emails (email required, name optional)
+ * @param int|string $emailTemplateId required
+ * @param array $variables optional
+ */
+$subscriber->sendTransactionalEmail($emails, $emailTemplateId, $variables)
+```
+
+### SYMFONY Implementation
+
+With symfony, you don't have to implement the factory,
+you can directly implement the interface and thus make the code generic,
+which means that if you want to change platform later,
+you just have to change the register implementation.
+
+```php
+class YourClass
+{
+    public function __construct(
+        private SubscriberInterface $subscriber
+    ) {
+    }
+    
+    public function sendTransactional()
+    {
+        $this->subscriber->sendTransactionalEmail(
+            [
+                new EmailAddress('user@example.com')
+            ],
+            $templateId
+        )
+    }
+}
+```
```

Comment:
```suggestion
```php
final class YourClass
{
    public function __construct(
        // Make it generic, let Symfony provide the right service for you
        private SubscriberInterface $subscriber
    ) {
    }
    
    public function sendTransactional()
    {
        $this->subscriber->sendTransactionalEmail(
            [
                new EmailAddress('user@example.com')
            ],
            $templateId
        )
    }
}
```
```